waltz-tools/src/test/java/com/wepay/waltz/tools/storage/StorageCliTest.java

```diff
@@ -370,9 +416,11 @@ public void testRemovePartitionWithDeleteFilesOption() throws Exception {
     @SuppressWarnings("OptionalGetWithoutIsPresent")
     @Test
     public void testAvailability() throws Exception {
-        int numPartitions = 3;
+        int numPartitions = 5;
         IntegrationTestHelper helper = getIntegrationTestHelper(numPartitions);
-        int partitionId = new Random().nextInt(helper.getNumPartitions());
+        int partitionId1 = new Random().nextInt(helper.getNumPartitions());
+        // ensure partitionId2 is different from partitionId1
+        int partitionId2 = (partitionId1 + 1 + new Random().nextInt(helper.getNumPartitions() - 2)) % helper.getNumPartitions();
```

Comment:
Same here. "(partitionId1 + 1) % helper.getNumPartitions()" should be good enough